core/ui/src/views/DomainConfiguration.vue

```diff
@@ -133,82 +133,85 @@
+                <div class="provider-card-content">
+                  <div class="row">
+                    <span class="label right-margin">{{
+                      $t("domains.expiration")
+                    }}</span>
 
-                  <cv-tag
-                    v-if="policy.expiration.enforced"
-                    kind="green"
-                    :label="$t('common.enabled')"
-                    size="sm"
-                    class="no-margin"
-                  ></cv-tag>
-                  <cv-tag
-                    v-else
-                    kind="high-contrast"
-                    :label="$t('common.disabled')"
-                    size="sm"
-                    class="no-margin"
-                  ></cv-tag>
-                </div>
-                <div class="row">
-                  <span class="label right-margin">{{
-                    $t("domains.strength")
-                  }}</span>
-                  <cv-tag
-                    v-if="policy.strength.enforced"
-                    kind="green"
-                    :label="$t('common.enabled')"
-                    size="sm"
-                  ></cv-tag>
-                  <cv-tag
-                    v-else
-                    kind="high-contrast"
-                    :label="$t('common.disabled')"
-                    size="sm"
-                  ></cv-tag>
+                    <cv-tag
+                      v-if="policy.expiration.enforced"
+                      kind="green"
+                      :label="$t('common.enabled')"
+                      size="sm"
+                      class="no-margin"
+                    ></cv-tag>
+                    <cv-tag
+                      v-else
+                      kind="high-contrast"
+                      :label="$t('common.disabled')"
+                      size="sm"
+                      class="no-margin"
+                    ></cv-tag>
+                  </div>
+                  <div class="row">
+                    <span class="label right-margin">{{
+                      $t("domains.strength")
+                    }}</span>
+                    <cv-tag
+                      v-if="policy.strength.enforced"
+                      kind="green"
+                      :label="$t('common.enabled')"
+                      size="sm"
+                    ></cv-tag>
+                    <cv-tag
+                      v-else
+                      kind="high-contrast"
+                      :label="$t('common.disabled')"
+                      size="sm"
+                    ></cv-tag>
+                  </div>
                 </div>
-              </div>
-            </template>
-          </NsInfoCard>
-        </cv-column>
+              </template>
+            </NsInfoCard>
+          </cv-column>
+        </template>
         <!-- user portal link -->
         <cv-column :md="4" :max="4">
           <NsInfoCard
             light
+            v-if="domain.location == 'internal'"
```

Comment:
Let's put this `v-if` into the `<cv-column>` element

Reply:
done